charts/vault/values/plugin-secrets-kubernetes-reader.yaml

```diff
@@ -0,0 +1,34 @@
+server:
+  dev:
+    enabled: true
+  extraArgs: '-dev-root-token-id=root -dev-plugin-dir=/usr/local/libexec/vault'
```

Comment:
What does it do?

Reply:
dev-root-token-id sets the vault token to root and dev-plugin-dir registers all plugin under the plugin directory when running in dev mode. in non dev mode there are other steps to register the plugin.

Reply:
Can you please share a link to the vault docs?

Reply:
Sure: https://www.vaultproject.io/docs/commands/server#dev-options